app/src/main/java/com/nextcloud/talk/ui/dialog/MessageActionsDialog.kt

```diff
@@ -82,7 +83,8 @@ class MessageActionsDialog(
         )
         initMenuDeleteMessage(showMessageDeletionButton)
         initMenuForwardMessage(
-            ChatMessage.MessageType.REGULAR_TEXT_MESSAGE == message.getMessageType() &&
+            hasChatPermission &&
```

Comment:
wouldn't forward be something ending up in another chat and thus not be blocked by this chat permission, since the permission is set on the room?

Reply:
see commit message https://github.com/nextcloud/talk-android/pull/2019/commits/b603031011c5fef7f1bfceb6f275e2fde0017566

but tbh i don't get the point anymore why i had this opinion. will check again...